docs/source/recipes/creating_views.ipynb

```diff
@@ -1,1140 +1,1140 @@
+          ]
+        }
+      ],
+      "source": [
+        "print(dataset.distinct(\"predictions.detections.label\"))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SsW1p1hVlikB"
+      },
+      "source": [
+        "However, you can also pass a [ViewExpression](https://voxel51.com/docs/fiftyone/api/fiftyone.core.expressions.html#fiftyone.core.expressions.ViewExpression) to the aggregation method, in which case the expression will be evaluated and then aggregated as requested:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "2PVf7k_RlxPJ",
+        "outputId": "14aaed91-5769-4f36-a418-26157c7c00f5"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "[0.15, 0.16, 0.17, 0.18, 0.19, 0.2, 0.21, 0.22, 0.23, 0.24, 0.25, 0.26, 0.27, 0.28, 0.29, 0.34, 0.53, 0.54, 0.55, 0.56, 0.57, 0.58, 0.59, 0.6, 0.61, 0.62, 0.63, 0.64, 0.65, 0.66, 0.67, 0.68, 0.69, 0.7, 0.72, 0.73, 0.74, 0.75, 0.78, 0.8, 0.82, 0.92, 1.0]\n"
+          ]
+        }
+      ],
+      "source": [
+        "print(dataset.distinct(F(\"uniqueness\").round(2)))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "T3dejQsznenG"
+      },
+      "source": [
+        "## Summary\n",
+        "\n",
+        "Dataset views and the view expressions language are powerful and flexible aspects of FiftyOne.\n",
+        "\n",
+        "Getting comfortable with using views and expressions to slice and dice your datasets based on the questions you have will allow you to work efficiently to curate high quality datasets."
+      ]
     }
-   ],
-   "source": [
-    "print(dataset.distinct(\"predictions.detections.label\"))"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "id": "SsW1p1hVlikB"
-   },
-   "source": [
-    "However, you can also pass a [ViewExpression](https://voxel51.com/docs/fiftyone/api/fiftyone.core.expressions.html#fiftyone.core.expressions.ViewExpression) to the aggregation method, in which case the expression will be evaluated and then aggregated as requested:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
+  ],
+  "metadata": {
     "colab": {
-     "base_uri": "https://localhost:8080/"
+      "collapsed_sections": [],
+      "name": "FiftyOne_Enterprise_View_Expressions.ipynb",
```

Comment:
doesnt seem right

Reply:
I don't know what happened here. This file got heavily edited unintentionally based on the diff, I just restored it back to the base release branch

Reply:
the .ipynb files might need to be manually inspected and edited tbh